pyproject.toml

```diff
@@ -358,10 +358,6 @@ lint.ignore = [  # NOTE: non-permanent exclusions should be added to `.ruff.toml
     # flake8-fixme (FIX)
     "FIX002",  # Line contains TODO | notes for improvements are OK iff the code works
 
-    # ISC001 shouldn't be used with ruff format
-    # https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules
-    "ISC001",
```

Comment:
The comment used to be true, but the incompatibility was solved in [Ruff 0.9.0](https://github.com/astral-sh/ruff/releases/tag/0.9.0).